tests/features/solution/solution_overview.feature

```diff
@@ -0,0 +1,101 @@
+@api
+Feature: Solutions Overview
+
+  Scenario: Check visibility of "Solutions" menu link.
+    Given I am an anonymous user
+    Then I should see the link "Solutions"
+    When I click "Solutions"
+    Then I should see the heading "Solutions"
+    # Check that all logged in users can see and access the link as well.
+    Given I am logged in as a user with the "authenticated user" role
+    Then I should see the link "Solutions"
+    When I click "Solutions"
+    Then I should see the heading "Solutions"
+
+  # @todo The small header, which contains solutions link, should be removed for anonymous users on the homepage - https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-2639.
+  @terms
+  Scenario: View solution overview as an anonymous user
+    Given users:
+      | name          | mail                              | roles |
+      | Madam Shirley | i.dont.see.the.future@example.com |       |
+    Given solutions:
+      | title                 | description                    | state     |
+      | Non electronic health | Supports health-related fields | validated |
+      | Closed data           | Facilitate access to data sets | validated |
+      | Isolating Europe      | Reusable tools and services    | validated |
+    Given the following owner:
+      | name              | type                    |
+      | NonProfit example | Non-Profit Organisation |
+    And the following contact:
+      | email | foo@bar.com         |
+      | name  | Information example |
+    # Check that visiting as an anonymous does not create cache for all users.
```

Comment:
In the content test I added the moderator as the first user to visit the page. Do the same here.